### PR TITLE
hwdb: update for v257

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

File: tools/chromiumos/gen_autosuspend_rules.py [hunk 1]
```diff
@@ -1,10 +1,7 @@
 #!/usr/bin/env python3
-# SPDX-License-Identifier: BSD-3-Clause
```
Comment (bluca) [inline tools/chromiumos/gen_autosuspend_rules.py]: you need to manually drop these changes to the header in this file before committing

Comment (poettering) [reply tools/chromiumos/gen_autosuspend_rules.py]: why?

Comment (bluca) [reply tools/chromiumos/gen_autosuspend_rules.py]: Because it drops the SPDX line and refers to a license file that doesn't exist

Comment (bluca) [reply tools/chromiumos/gen_autosuspend_rules.py]: fixed it myself so we can merge this

**Pull request merged**